test/e2e_node/services/util.go

```diff
@@ -17,19 +17,33 @@ limitations under the License.
 package services
 
 import (
+	"crypto/tls"
 	"fmt"
-	"k8s.io/klog/v2"
 	"net/http"
 	"os"
 	"os/signal"
 	"syscall"
 	"time"
+
+	"k8s.io/klog/v2"
+
+	"k8s.io/kubernetes/test/e2e/framework"
 )
 
 // terminationSignals are signals that cause the program to exit in the
 // supported platforms (linux, darwin, windows).
 var terminationSignals = []os.Signal{syscall.SIGHUP, syscall.SIGINT, syscall.SIGTERM, syscall.SIGQUIT}
 
+var insecureHTTPClient = &http.Client{}
+
+const tokenFilePath = "known_tokens.csv"
+
+func init() {
```

Comment:
Can we avoid this init function - e.g. make the client local to functions that need it?